data/costs.csv

```diff
@@ -1,195 +1,195 @@
-decentral water tank storage,2030,FOM,1,%/year,HP
-decentral water tank storage,2030,lifetime,20,years,HP
-decentral water tank storage,2030,discount rate,0.04,per unit,Palzer thesis
-central water tank storage,2030,investment,30,EUR/m3,IWES Interaktion
-central water tank storage,2030,FOM,1,%/year,HP
-central water tank storage,2030,lifetime,40,years,HP
-decentral resistive heater,2030,investment,100,EUR/kWhth,Schaber thesis
-decentral resistive heater,2030,lifetime,20,years,Schaber thesis
-decentral resistive heater,2030,FOM,2,%/year,Schaber thesis
-decentral resistive heater,2030,efficiency,0.9,per unit,Schaber thesis
-decentral resistive heater,2030,discount rate,0.04,per unit,Palzer thesis
-central resistive heater,2030,investment,100,EUR/kWhth,Schaber thesis
-central resistive heater,2030,lifetime,20,years,Schaber thesis
-central resistive heater,2030,FOM,2,%/year,Schaber thesis
-central resistive heater,2030,efficiency,0.9,per unit,Schaber thesis
-decentral gas boiler,2030,investment,175,EUR/kWhth,Palzer thesis
-decentral gas boiler,2030,lifetime,20,years,Palzer thesis
-decentral gas boiler,2030,FOM,2,%/year,Palzer thesis
-decentral gas boiler,2030,efficiency,0.9,per unit,Palzer thesis
-decentral gas boiler,2030,discount rate,0.04,per unit,Palzer thesis
-central gas boiler,2030,investment,63,EUR/kWhth,Palzer thesis
-central gas boiler,2030,lifetime,22,years,Palzer thesis
-central gas boiler,2030,FOM,1,%/year,Palzer thesis
-central gas boiler,2030,efficiency,0.9,per unit,Palzer thesis
-decentral CHP,2030,lifetime,25,years,HP
-decentral CHP,2030,investment,1400,EUR/kWel,HP
-decentral CHP,2030,FOM,3,%/year,HP
-decentral CHP,2030,discount rate,0.04,per unit,Palzer thesis
-central CHP,2030,lifetime,25,years,HP
-central CHP,2030,investment,650,EUR/kWel,HP
-central CHP,2030,FOM,3,%/year,HP
-decentral solar thermal,2030,discount rate,0.04,per unit,Palzer thesis
-decentral solar thermal,2030,FOM,1.3,%/year,HP
-decentral solar thermal,2030,investment,270000,EUR/1000m2,HP
-decentral solar thermal,2030,lifetime,20,years,HP
-central solar thermal,2030,FOM,1.4,%/year,HP
-central solar thermal,2030,investment,140000,EUR/1000m2,HP
-central solar thermal,2030,lifetime,20,years,HP
-HVAC overhead,2030,investment,400,EUR/MW/km,Hagspiel
-HVAC overhead,2030,lifetime,40,years,Hagspiel
-HVAC overhead,2030,FOM,2,%/year,Hagspiel
-HVDC overhead,2030,investment,400,EUR/MW/km,Hagspiel
-HVDC overhead,2030,lifetime,40,years,Hagspiel
-HVDC overhead,2030,FOM,2,%/year,Hagspiel
-HVDC submarine,2030,investment,2000,EUR/MW/km,DTU report based on Fig 34 of https://ec.europa.eu/energy/sites/ener/files/documents/2014_nsog_report.pdf
-HVDC submarine,2030,lifetime,40,years,Hagspiel
-HVDC submarine,2030,FOM,2,%/year,Hagspiel
-HVDC inverter pair,2030,investment,150000,EUR/MW,Hagspiel
-HVDC inverter pair,2030,lifetime,40,years,Hagspiel
-HVDC inverter pair,2030,FOM,2,%/year,Hagspiel
+technology,year,parameter,value,unit,source

```

Comment:
What about this csv change? is there a reason?

In general, let's avoid to commit changes in binaries, unless needed for some reason

Reply:
Not intentional.. that happened I believe when I stopped a model run and started again with --rerun-incomplete flag.

Reply:
Ok! are there changes or is it the same file?

Reply:
I did not change any values. Should be the same file